src/util/cert/cert.go

```diff
@@ -117,7 +117,7 @@ func certKeyXor(certFile, keyFile string) (bool, error) {
 		return false, fmt.Errorf("Key %s is not a file", keyFile)
 	default:
 		panic("unreachable code")
-		return false, fmt.Errorf("unreachable code")
+		//return false, fmt.Errorf("unreachable code")
```

Comment:
@gz-c , this code is unreachable.
Got vet error:
vendorcheck ./...
gometalinter --disable-all --enable=vet -E goimports --tests --vendor ./...
src/util/cert/cert.go:120::error: unreachable code (vet)
make: *** [lint] Error 1

Reply:
Delete it